libraries/Microsoft.Bot.Builder.Azure/AzureTableStorage.cs

```diff
@@ -107,15 +107,22 @@ public AzureTableStorage(CloudStorageAccount storageAccount, string tableName)
                 return new KeyValuePair<string, object>();
             });
 
-            return (await Task.WhenAll(readTasks).ConfigureAwait(false))
-                .Where(kv => kv.Key != null);
+            var kvps = (await Task.WhenAll(readTasks).ConfigureAwait(false)).Where(kv => kv.Key != null);
+
+            IDictionary<string, object> items = new Dictionary<string, object>();
```

Comment:
The codebase seems to be `var` based, so I think we would want to stay consistent with that.